lib/flagutil/duration.go

```diff
@@ -12,14 +12,14 @@ import (
 // NewDuration returns new `duration` flag with the given name, defaultValue and description.
 //
 // DefaultValue is in months.
-func NewDuration(name string, defaultValue float64, description string) *Duration {
+func NewDuration(name string, defaultValue string, description string) *Duration {
```

Comment:
I don't think, that we should change API for exist flag or it should be done via separate PR.